src/Shared/Microsoft.DotNet.Scaffolding.Shared/Cli.Utils/DotnetCommands.cs

```diff
@@ -55,7 +55,7 @@ public static void ExecuteDotnetNew(string projectPath, IList<string> additional
             {
                 validDotnetVersion = parsedVersion.CompareTo(MinimumDotnetVersion) >= 0;
                 var validText = validDotnetVersion ? "meets" : "does not meet";
-                consoleLogger.LogMessage($"Found dotnet version ({parsedVersion}). It {validText} the minimum requirement.\n");
+                consoleLogger.LogMessage($"\nFound dotnet version ({parsedVersion}). It {validText} the minimum requirement.\n");
```

Comment:
I'd define different, separated message for the 2 cases: as it is, it is not translatable at all, and is not a good practice either.

Reply:
Ok, changed logging to only logging if requirements are not met, added error message to MessageStrings.resx